src/sas/qtgui/Utilities/GuiUtils.py

```diff
@@ -32,12 +32,14 @@
 from sasdata.dataloader.data_info import Sample, Source, Vector
 from sasdata.dataloader.data_info import Detector, Process, TransmissionSpectrum
 from sasdata.dataloader.data_info import Aperture, Collimation
+from sas.sascalc.doc_regen.makedocumentation import HELP_DIRECTORY_LOCATION
```

Comment:
```suggestion
from sas.sascalc.doc_regen.makedocumentation import HELP_DIRECTORY_LOCATION, PATH_LIKE
```
This will allow you to use PATH_LIKE for type of `showHelp`